### ISSUE TITLE
New blocks missing from RPC Daemon

### ISSUE THREAD

**Issue by iFA88** [ledgerwatch/erigon#5671]
Hey!

Sadly there is lot of notification missing during syncing and so new block are not visible on query, like the sync has stuck.
Version: `git_tag=v2.27.0 git_commit=6d8b304eae78cef7aa3f3eb7ed584559bdfe7dbb`
I excepted that after finalize the blocks are visible for the RPC.

Last notifications:
```
[INFO] [10-08|11:55:57.339] RPC Daemon notified of new headers       from=2045055 to=2045157
[INFO] [10-08|13:55:18.380] RPC Daemon notified of new headers       from=2045157 to=2045728
```

Whole log:
https://www.fusionsolutions.io/doc/tethNode_2022-10-07_16-55.log

Thanks for the support!

**Comment by iFA88** [comment]
Issue still available with `v2.28.0`

**Comment by zekchan** [comment]
Same error, I think it is connected with https://github.com/ledgerwatch/erigon/issues/5716
